superset/models/slice.py

```diff
@@ -55,7 +55,7 @@
 
 class Slice(
     Model, AuditMixinNullable, ImportExportMixin
-):  # pylint: disable=too-many-public-methods
+):  # pylint: disable=too-many-public-methods, too-many-instance-attributes
```

Comment:
we could just update the max attributes
https://github.com/apache/superset/blob/master/.pylintrc#L385

Reply:
Hmmmm, I'm not sure I want to change the settings for the whole codebase. I like being conservative.